src/bin/restore.js

```diff
@@ -42,8 +43,19 @@ if (compilerPlugins) {
   processor.compilerPlugins.push(...requireByPath(compilerPlugins));
 }
 
+var check = false
+if (argv.v || argv.checkRestorations) {
+  check = true;
+}
+
+function restore_with_check() {
```

Comment:
why is this necessary? Can't we just do `recursivelyProcessAll(processor.sourceAndRedactedToRestored.bind(processor, check), data);`?